### PR TITLE
Change sacra image and remove parent protocol

### PR DESCRIPTION
**Pull request by alexandersazonof**

### CODE REVIEW

File: defi/src/protocols/data3.ts [hunk 1]
```diff
@@ -62195,7 +62195,6 @@ const data3: Protocol[] = [
     twitter: "sacra_fi",
     forkedFrom: [],
     oracles: [],
-    parentProtocol: "parent#tetu",
```
Comment (Define101) [inline defi/src/protocols/data3.ts:62198]: Why remove this?

Comment (alexandersazonof) [reply defi/src/protocols/data3.ts:62198]: Because its different projects

Comment (alexandersazonof) [reply defi/src/protocols/data3.ts:62198]: @Define101 I can return if its important

Comment (realdealshaman) [reply defi/src/protocols/data3.ts:62198]: @alexandersazonof we also group projects together when they are part of the same team. I notice that they are all being worked on from the tetu github repo

Comment (alexandersazonof) [reply defi/src/protocols/data3.ts:62198]: return parent protocol

Comment (alexandersazonof) [reply defi/src/protocols/data3.ts:62198]: @realdealshaman ping

**Pull request merged**